src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -203,7 +199,8 @@ private BindingConfiguration QueueWriteBindingInformation()
 
         public void PrepareToInstallPackages()
         {
-            this.NuGetBindingOperation.PrepareOnUIThread();
+            // TODO - CM cleanup
+            // no-op
```

Comment:
A long time ago, the original binding process used to install the C#/VB.NET analyzers as NuGet packages. This code was kept for backwards compatibility, and we can drop it now.

The NuGet installation was a two-step process: "prepare" then "install".
I've made the minimal changes to `BindingProcessImpl` again - just hard-coded the methods do nothing/return success.